modules/claims_api/config/routes.rb

```diff
@@ -49,7 +49,7 @@
             ClaimsApi::V2::Veterans::IntentToFilesController::ITF_TYPES.include?(request.path_parameters[:type])
           end)
       post '/:veteranId/intent-to-files', to: 'intent_to_files#submit'
-      post '/:veteranId/5103', to: 'evidence_waiver#submit'
+      post '/:veteranId/5103/:claimId', to: 'evidence_waiver#submit'
```

Comment:
still thinking this through. I'm leaning towards an end goal of something like 
`veterans/:veteranId/claims/:claimId/5103` or something similar... matching v1 style of submissions it could even be `veterans/:veteranId/claims/:claimId/forms/5103`. We'd want to rework the routes file with some namespaces to achieve this and nest a few items.

Reply:
this being said, it is all preluded with `/services/claims` so I wonder if duplicating `claims` would be good? Give me a bit to think on this.

Reply:
[3668e4d](https://github.com/department-of-veterans-affairs/vets-api/pull/10822/commits/3668e4d2351c39c944b9c2b47909570776df1b53)